components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -1432,6 +1435,12 @@ public static String createEndpoint(String endpointUrl, Registry registry) throw
             String msg = "Malformed XML found in the External Stores Configuration resource";
             log.error(msg, e);
             throw new APIManagementException(msg, e);
+        } catch (ClassNotFoundException e) {
```

Comment:
If external Store class not in the Class Path Then 1388 line is throws class not found exception and this class can be any class that extend APIPublisher this will be changed from external store